### ISSUE TITLE
[BUG]ComputerCraft-tweaked

### ISSUE THREAD

**Issue by mozomenku** [magmafoundation/Magma#251]
**Describe the bug**
A clear and concise description of what the bug is: Server can't reload due to some problem with CC-tweaked

**Expected behavior**
A clear and concise description of what you expected to happen: Server should reload with /reload command but it gives errors

**Info (please complete the following information):**
- Mod Name: ComputerCraft
 - Mod Version: 1.12.2-1.89.2
 - Magma Version: ade7751
 - OS: Debian

**Additional context**
Add any other context about the problem here: https://pastebin.com/DQev1NX3

**Comment by mozomenku** [comment]
I have worldedit mod and it works with that.

**Comment by Amgelo563** [comment]
Never use `/reload`, it's too broken even on Spigot/Paper servers.

**Comment by mozomenku** [comment]
So just restart the server?

**Comment by Amgelo563** [comment]
Yes.

**Comment by Frerduro** [comment]
TLDR never /reload always restart
